modules/k3s/docker/variables.tf

```diff
@@ -21,3 +21,16 @@ variable "cluster_endpoint" {
   type        = string
   default     = null
 }
+
+variable "worker_groups" {
+  description = "A list of maps defining worker group configurations"
+  type        = map(any)
```

Comment:
Could we describe the type instead of any? Like

```
type = map(object({
node_count = number
node_labels = list(string)
node_taints = list(string)
}))

Reply:
Probably better indeed.